crates/autopilot/src/database/order_events.rs

```diff
@@ -18,21 +18,19 @@ impl super::Postgres {
 }
 
 pub async fn store_order_events(
-    db: &super::Postgres,
+    ex: &mut PgConnection,
     order_uids: Vec<domain::OrderUid>,
     label: OrderEventLabel,
     timestamp: DateTime<Utc>,
 ) -> Result<()> {
-    let mut ex = db.pool.begin().await.context("begin transaction")?;
```

Comment:
What's the reason to no longer build a transaction here?

Reply:
becuase in order to make this function usable outside the `persistence` module I needed to pass to it the `ex: &mut PgConnection` already, so it is not needed to begin the tx.

Reply:
btw, why don't we have the database access unified within the `autopilot`? is there any reason for it?